<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Remote</class>
 <widget class="QWidget" name="Remote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Remote {
	background-color:qlineargradient(spread:pad, x1:0.174, y1:0.187, x2:0.965, y2:1, stop:0.00497512 rgba(202, 255, 221, 255), stop:0.338308 rgba(98, 228, 168, 255), stop:0.706468 rgba(144, 179, 255, 255), stop:1 rgba(108, 231, 255, 255));
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="80,100,200,100">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="headerLayout" rowstretch="60,20" columnstretch="80,720" columnminimumwidth="0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0" rowspan="2">
      <widget class="QPushButton" name="pb_back">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">#pb_back {
border-right: 4px solid black;
border-bottom: 4px solid black;
border-bottom-right-radius: 16px;
background-color:blue;
}

#pb_back:pressed {
border-right: 8px solid black;
border-bottom: 8px solid black;
}

#pb_back::disabled {
background-color: grey;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/images/back.png</normaloff>:/images/back.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="title">
       <property name="styleSheet">
        <string notr="true">#title {
	background-color: rgb(74, 70, 122);
	border-bottom: 1px solid black;
	color: white;
}</string>
       </property>
       <property name="text">
        <string>Сервер для удаленного запуска</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="3,1" columnstretch="1,1,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pb_server">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">#pb_server {
border: 4px solid black;
border-radius: 16px;
background-color: green;
color: white;
}

#pb_server::checked {
background-color: red;
}

#pb_server::disabled {
background-color: grey;
}

#pb_server:pressed {
border: 8px solid black;
}</string>
       </property>
       <property name="text">
        <string>Отслеживать
сообщения
сервера</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="lbl_status">
       <property name="text">
        <string>Статус: сообщения не отслеживаются</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="indent">
        <number>80</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
